src/fiat/bank-authorities.md

```diff
@@ -106,14 +108,39 @@ curl -X GET https://service.centrapay.com/api/bank-authorities/Jaim1Cu1Q55uooxSe
     "id": "WRhAxxWpTKb5U7pXyxQjjY",
     "accountId": "Jaim1Cu1Q55uooxSens6yk",
     "fullName": "John Doe",
-    "streetAddress": "11 Shortland St",
-    "suburb": "Auckland Central",
-    "city": "Auckland",
-    "postCode": "1100",
-    "phoneNumber": "+64212345678",
-    "emailAddress": "John.doe@email.com",
     "bankAccountNumber": "12-1234-1234567-123",
     "bankAccountName": "John Doe",
+    "topupFieldsRequired": [ "userAuthorizesDirectDebit" ],
+    "status": "created",
+    "verified": false,
+    "createdAt": "2020-06-12T01:17:46.499Z",
+    "createdBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+    "modifiedAt": "2020-06-12T01:17:46.499Z",
+    "modifiedBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+}
+```
+
+## Update information about a bank authority **EXPERIMENTAL**
+
+{% endpoint PUT https://service.centrapay.com/api/bank-authorities/${id} %}
+
+```sh
+curl -X PUT https://service.centrapay.com/api/bank-authorities/Jaim1Cu1Q55uooxSens6yk \
+  -H "x-api-key: 1234" \
+  -H "content-type: application/json" \
+  -d '{ "userAuthorizesDirectDebit": true }'
```

Comment:
We will need to update the full name and email address too right?

Reply:
I'll add some documentation around the fields, good catch!

Reply:
Added docs with additional fields 📚

Reply:
I don't get why this is necessary